### PR TITLE
Steffi's Review

### PR DESCRIPTION
**Pull request by steffilazerte**
Here is where I'll be adding commits related to my review of this guide. 

I will also be using issues (one per chapter) to track more general comments. I'll update this PR as I make my way through the book.

### Progress
- [X] Home page (Comments #53)
- [X] 1. Motivation (No Comments)
- [x] 2. General Guidelines
- [x] 3. Translation and Review Guidelines
- [x] 4. Specific Guidelines

### To do
- [x] Run general spell-checker
- [x] Run general URL checker
- [x] Make links clickable

### CODE REVIEW

**Comment by steffilazerte** [comment]
Hi @yabellini, I'm done! I made many direct changes to the text. You can see some of my rational and other suggestions as well as other commetns adn details in the issues, and we can discuss in our next meeting too. 

- #53 
- #54 
- #55 
- #56

File: index.Rmd [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
 
 Ana
 :  
-  is a Colombian student doing her Ph.D. at an university in Argentina. She works with geospatial data. She is not an expert in programming, but she took one of The [Carpentries](https://carpentries.org/) workshops for scientists and used some of the rOpenSci suite packages for handling and accessing geospatial data for her thesis.
+  is a Colombian student doing her Ph.D. at an university in Argentina. She works with geospatial data. She is not an expert in programming, but she took one of [The Carpentries](https://carpentries.org/) workshops for scientists and used some of the rOpenSci suite packages for handling and accessing geospatial data for her thesis.
   Language was one of the barriers she faced when learning to code. Ana wants to contribute by translating material into her native language so others can access this knowledge with less effort. This book will show her how to do a new translation and how to review an existing one.
```
Comment (yabellini) [inline index.Rmd]:
```suggestion
  Language was one of the barriers she faced when learning to code. Ana wants to contribute by translating rOpenSci material into her native language so others can access this knowledge with less effort. This book will show her how to do a new translation and how to review an existing one.
```

File: howtoreview.Rmd [hunk 2]
```diff
@@ -1,119 +1,154 @@
...
+whereas the review of the Spanish translation would happen on the `pkg_ci.es.Rmd-es-auto` branch.
 
-To start contributing the first thing you should do is to clone the working repository.
-To find out where it is located, talk to the person in charge of the project or consult the Slack channel.
-Once you have the work repository cloned, you can start working locally on your computer.
+Once you've [contacted the project maintainer](#contributing) and you're ready to start contributing, 
+the first thing you should do is to clone the working repository.
+Once cloned, you can start working with the repository locally on your computer.
+If you haven't already had a discussion about what you will be reviewing, 
+you can ask in the Slack channel or comment on a *pull request* to ask if you can take the review of that chapter.
```
Comment (yabellini) [inline howtoreview.Rmd:70]:
```suggestion
you can ask in the Slack channel or comment on a *pull request* to ask if you can take the review of that chapter.
We strongly suggest that you ask **before** starting work on your review. 
```

File: motivation.Rmd [hunk 3]
```diff
@@ -76,15 +77,15 @@ Spanish is the second most-spoken native language in the world and is one of the
 
 [^motivation-5]: List of languages by number of native speakers.
     Accessed on December 1, 2022.
-    https://en.wikipedia.org/wiki/List_of_languages_by_number_of_native_speakers
+    <https://en.wikipedia.org/wiki/List_of_languages_by_number_of_native_speakers>
 
 ## Building community
 
-At rOpenSci, we understand review as a way of building community and have our review process in Spanish will allow us to continue building the community in Spanish-speaking regions, to increase the number of contributors, and to get feedback on how our tools and processes can be improved to better serve these community members.
+At rOpenSci, we understand review as a way of building community and hope our review process in Spanish will allow us to continue building the community in Spanish-speaking regions, to increase the number of contributors, and to get feedback on how our tools and processes can be improved to better serve these community members.
```
Comment (yabellini) [inline motivation.Rmd]:
```suggestion
At rOpenSci, we understand review as a way of building community and hope our review process in Spanish and Portuguese, will allow us to continue building the community in regions that speak these languages, to increase the number of contributors, and to get feedback on how our tools and processes can be improved to better serve these community members.
```

File: specific_guidelines.Rmd [hunk 4]
```diff
@@ -1,33 +1,27 @@
...
-## Aspects to take into account in the translation of the text
 
-__1. Use a conversational voice instead of a formal or academic voice.__
-
-__2. When translating, we seek to generate a clear version in English of what is written in the original language in the chapter.__ 
-
-__3. The dialectic variety of English that we will use in the translation is American English.__
-
-__4. Idiomatic expressions are not translatable in a literal way.__
+[[STEFFI SUGGESTS MOVING THE FOLLOWING POINT TO CHP 3]]
```
Comment (yabellini) [inline specific_guidelines.Rmd:16]: Agree is the best place.

**Review by yabellini — APPROVED** [review]
Thank you Steffi!  I will merge your suggestions and then I will review the issues to see what changes we can incorporate in this first version and what we leave for the second version. 

As always, the content is much clearer and flows better thanks to your editing.

**Pull request merged**